.github/workflows/subgraph-deployment.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'subgraphs/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: false
+
+jobs:
+  deploy-subgraph:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bloq/actions/notify-deploy-to-slack@v1
+        with:
+          app-name: Hemi Portal Subgraphs
+          environment: staging
+          reference: ${{ github.event.head_commit.message }}
+          slack-mention: ${{ vars.SLACK_MENTION }}
+          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          status: 'started :stopwatch:'
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+      - uses: hemilabs/actions/setup-node-env@v1
+      - run: npx --yes graph auth "$GRAPH_API_KEY"
+        env:
+          GRAPH_API_KEY: ${{ secrets.GRAPH_API_KEY }}
+      - id: version-check
+        run: |
+          git fetch origin main
+          SUBGRAPH_NAME="${{ matrix.subgraph }}"
+          PREVIOUS_VERSION=$(git show origin/main:subgraphs/${SUBGRAPH_NAME}/package.json | jq -r .version 2>/dev/null || echo "none")
+          CURRENT_VERSION=$(jq -r .version subgraphs/${SUBGRAPH_NAME}/package.json)
+          if [ "$PREVIOUS_VERSION" = "none" || "$CURRENT_VERSION" != "$PREVIOUS_VERSION" ]; then
+            echo "VERSION_CHANGED=true" >> $GITHUB_OUTPUT
+          else
+            echo "VERSION_CHANGED=false" >> $GITHUB_OUTPUT
+          fi
+      - if: steps.version-check.outputs.VERSION_CHANGED == 'true'
+        run: npm run deploy
+        working-directory: subgraphs/${{ matrix.subgraph }}
+        continue-on-error: true
+      - if: ${{ !cancelled() }}
+        uses: bloq/actions/notify-deploy-to-slack@v1
+        with:
+          app-name: Hemi Portal Subgraphs
+          environment: staging
+          slack-mention: ${{ vars.SLACK_MENTION }}
+          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          status: ${{ job.status == 'failure' && 'failed :boom:' || 'finished :ok:' }}
```

Comment:
if a subgraph deployment fails, but others succeeded, do we get notified? I'm asking due to the ` continue-on-error: true`. I wouldn't want to get deployment failures swallowed!